test/integration/multikueue/jobs_test.go

```diff
@@ -308,6 +308,7 @@ var _ = ginkgo.Describe("MultiKueue", ginkgo.Ordered, ginkgo.ContinueOnFailure,
 
 	ginkgo.It("Should run a job on worker if admitted (ManagedBy)", func() {
 		features.SetFeatureGateDuringTest(ginkgo.GinkgoTB(), features.MultiKueueBatchJobWithManagedBy, true)
+		features.SetFeatureGateDuringTest(ginkgo.GinkgoTB(), features.PropagateBatchJobLabelsToWorkload, true)
```

Comment:
Revert, as mentioned in https://github.com/kubernetes-sigs/kueue/pull/7613/files#r2523322303, let's have a test in the provReq suite which corresponds to the business use case for the issue

EDIT: btw this is no-op as the feature is enabled by default anyway